spring-cloud-gcp-pubsub-stream-binder/src/main/java/com/google/cloud/spring/stream/binder/pubsub/provisioning/PubSubChannelProvisioner.java

```diff
@@ -79,7 +80,14 @@ public ConsumerDestination provisionConsumerDestination(String topicName, String
 
 		String subscriptionName;
 		Subscription subscription;
-		if (StringUtils.hasText(group)) {
+		if (StringUtils.hasText(properties.getExtension().getCustomSubscription())) {
+			String customName = properties.getExtension().getCustomSubscription();
+			subscriptionName = ProjectSubscriptionName.isParsableFrom(customName) ?
+					ProjectSubscriptionName.parse(customName).getSubscription() :
+					customName;
+			subscription = this.pubSubAdmin.getSubscription(customName);
+		}
+		else if (StringUtils.hasText(group)) {
```

Comment:
We should also log a warning if _both_ the custom subscription name and group name are specified. And document in refdoc which takes precedence.

Reply:
Done.